elements/pfe-navigation/README.md

```diff
@@ -6,14 +6,59 @@ This element will give you a top-level navigation element (not for use as subnav
 
 ### Slots
```

Comment:
Suggest adding this note at the top of the list:

All slots are optional and can be left off if not needed.  Slots prefixed with `mobile-`, however, do require their counterparts in order to render.